src/components/routineSingle.js

```diff
@@ -0,0 +1,27 @@
+const RoutineSingle = () => {
+
+
+
+
+    return (
+        <>
```

Comment:
It would be better to change the react fragment wrapping all the information into a div with a className='routine-single'. This will help you with styling the page later down the line.